api/src/test/java/io/opencensus/stats/ViewDataTest.java

```diff
@@ -171,10 +181,48 @@ public void preventStartTimeLaterThanEndTime() {
     CumulativeData.create(Timestamp.fromMillis(3000), Timestamp.fromMillis(2000));
   }
 
+  @Test
+  public void preventAggregationAndAggregationDataMismatch() {
+    aggregationAndAggregationDataMismatch(
+        createView(Sum.create(), MEASURE_DOUBLE),
+        ImmutableMap.<List<TagValue>, AggregationData>of(
+            Arrays.asList(V1, V2), SumDataLong.create(100)));
```

Comment:
I think each call to `aggregationAndAggregationDataMismatch` should be a separate test, since it throws an exception.

Reply:
Done.